### ISSUE TITLE
Insects in the ecosystem and agriculture

### ISSUE THREAD

**Issue by alexarthur94** [StrangeLoopGames/EcoSuggestions#652]
1. Bees would be a great addition to the game. They could be required for many types of plants to produce fruit and/or seeds. If bee populations are too low in an area, then fruit and/or seed production could be reduced substantially, and if they disappear completely, then those plants could be unable to produce fruit and/or seeds altogether, unless there is also a way to artificially pollinate them, which would be very time consuming. Once animal domestication is implemented, bees could also be domesticated on farms to improve crop yields. And bee hives, whether wild or domestic, could be used as a source of honey.
2. Insects that act as pests. In a natural ecosystem, they shouldn't be so populous as to cause any plant species a huge problem. However, in farms, they would reduce yield or maybe become so populous as to destroy a crop completely. Thus, pest control of some kind would be necessary. Synthetic pesticides that cause pollution would be one way, but they would also kill bees. Another way to control pests would be to plant companion plants that attract the pest's predator insects or to domestic predator insects, such as wasps and ladybugs. Or they could be controlled by rotating crops so that pest populations don't have enough time to increase too much. It would be fair for ecosystem-friendly means of pest control to be more laborious than synthetic pesticides since the work involved is the reason we use synthetic pesticides in real life.
3. All insects species should be necessary for the ecosystem's health, as well, as food for birds and predatory insects and/or as pollinators, depending on the species, and maybe even to control certain plant species that would otherwise act as weeds. 

Thanks for the awesome game! I know an addition like this might not come anytime soon, but I feel like this would be very cool.

**Comment by Kastuk** [comment]
For start can just let the herbivores to actually eat and kill their food sources plants.
So overpopulated deer will stomping your wheat fields.

**Comment by alexarthur94** [comment]
Agreed, as long as it's balanced so that a healthy wolf population is sufficient to control the deer population so the deer wouldn't damage the ecosystem.
